base/deprecated.jl

```diff
@@ -112,6 +112,18 @@ macro deprecate_binding(old, new, export_old=true)
          Expr(:call, :deprecate, __module__, Expr(:quote, old)))
 end
 
+macro deprecate_moved(old, new, export_old=true)
+    eold = esc(old)
+    return Expr(:toplevel,
+         :(function $eold(args...; kwargs...)
+               error($eold, args, " has been moved to the package ", $new, ".jl.\n",
```

Comment:
(not to mention that using `args` but not `kwargs` omits the keyword arguments 😛)